index.html

```diff
@@ -25,12 +25,14 @@
               src="SVG/hamburger-menu.svg"
               class="menuIcon hamburger"
               name="hamburger"
+              alt="menu"
             />
             <img
               src="SVG/cross.svg"
               class="menuIcon closebtn"
               id="cross"
               name="cross"
+              alt="menu"
             />
           </div>
           <div class="navlinks">
```

Comment:
- [OPTIONAL] I think it would be better to use semantic HTML5 tag `<nav>` instead of using `<div>` tag to follow the best practice and Improve the SEO👍

> For Example 💡
```html
<nav class="navlinks">
  .....
<nav
```

Reply:
ok, done